app/controllers/content.js

```diff
@@ -22,6 +22,12 @@ export default Ember.Controller.extend({
              this.set('authors', contributors));
     }),
 
+    applyMathJax: Ember.observer('model', function() {
```

Comment:
a) I'm not sure that's true- the model is the blank preprint, and some fields like DOI and subject are connected by two-way binding. (editing may happen even if saving does not)

b) Don't rely on a side effect of what you think will happen- be explicit.  If you only want something to fire once when the page loads, fire it in an appropriate event, not an observer. This will reduce the likelihood of future bugs, and as a bonus, will probably be more reliable for unit tests. (I've had problems in the past where unit tests of page rendering did not pick up on the observer in time, and moving the event to a rendering lifecycle hook fixed the failing test)